cfpb/consumerfinance.gov
# Update dependencies, including update to latest Design System

## Changes

- Update babel/core, babel/preset-env, autoprefixer, core-js, gulp-postcss, postcss, terser-webpack-plugin, webpack, webpack-stream, snyk.
- Update to 0.11.1 design system packages.
   - Updates to tagline from design system.
   - Fixes infinite filters bug (https://[ghe]/CFGOV/platform/issues/4082)
   - Brings in latest cards code.

## How to test this PR

1. Pull branch and run `./frontend.sh`
2. See tagline in header and footer. 
3. See change in cards.

## Notes and todos

- This changes the behavior of the tagline wrapping. We'll need to confirm with design before merging that we want this behavior at mobile:
<img width="325" alt="Screen Shot 2021-09-24 at 6 09 21 PM" src="https://user-images.githubusercontent.com/704760/134744722-04dcbaaf-80d8-4bb4-b23f-f867ee98f337.png">
<img width="567" alt="Screen Shot 2021-09-24 at 6 09 32 PM" src="https://user-images.githubusercontent.com/704760/134744724-a02376e8-324b-49c3-8e33-75b97bfc79e7.png">

<img width="477" alt="Screen Shot 2021-09-24 at 6 09 43 PM" src="https://user-images.githubusercontent.com/704760/134744727-4352c97c-e851-496b-ba8e-3875f97948bd.png">

- The current cards on the homepage are deprecated. The PR brings in the latest design system cards designs, which makes a slight change to the current cards. The links don't sit at the bottom of the cards anymore. These cards will be removed in the new homepage design.

<img width="1044" alt="Screen Shot 2021-09-24 at 6 09 59 PM" src="https://user-images.githubusercontent.com/704760/134744784-3065abd8-b33e-4ad1-bd02-d6518a408d14.png">
